components/placeholder/index.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import Dashicon from 'components/dashicon';
+
+function Placeholder( { icon, children, label, instructions, className, ...additionalProps } ) {
+	const classes = classnames( 'components-placeholder', className );
+
+	return (
+		<div { ...additionalProps } aria-label={ label } className={ classes }>
```

Comment:
If we don't feel we need it (and it doesn't appear we're using it?), we probably shouldn't encourage spreading additional props. We've done it elsewhere, and I find it's generally useful for generic wrapper components like a `<Button>`. But it comes at the cost of losing clarity to what's being passed to the element, and it's quite easy to pass unsupported props (for which React will log angrily).